### ISSUE TITLE
Name of page. Summary of issue

### ISSUE THREAD

**Issue by amazingphilippe** [cds-snc/notification-planning#1524]
## Description of issue

3.1.2: Language of Parts
AA

Describe what is failing

When content is bilingual, a lang attribute is missing to identify the language of content that is different than the page language. For example, on the Create Service Name and Email Address page, there is an example that contains text in both French and English. The French content is missing its own lang attribute. This could interfere with the correct pronunciation of content by screen readers. 

Users are also instructed to provide a bilingual name for the service, but the name appears in a single input. As a result the language is not differentiated between the French and English.

## Potential fix

- Have a way to encode language with user generated strings
- Make sure we display user generated strings with the encoded lang attribute. 

## Resources

- Links to documentation or articles that could be helpful
- examples: WCAG docs, ARIA docis, web.dev, Deque, Webaim...

**Comment by yaelberger-commits** [comment]
We could potentially reuse the pattern we created for logo alt-text here
